core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -353,7 +358,8 @@ class BrokerServer(
       dynamicConfigHandlers = Map[String, ConfigHandler](
         ConfigType.TOPIC -> new TopicConfigHandler(replicaManager, config, quotaManagers, None),
         ConfigType.BROKER -> new BrokerConfigHandler(config, quotaManagers),
-        ConfigType.CLIENT_METRICS -> new ClientMetricsConfigHandler(clientMetricsManager))
+        ConfigType.CLIENT_METRICS -> new ClientMetricsConfigHandler(clientMetricsManager),
+        ConfigType.GROUP -> new ConsumerGroupConfigHandler(consumerGroupConfigManager))
```

Comment:
`GroupConfigHandler` and `groupConfigManager`

Reply:
Done